manager/director/apps/sites/forms.py

```diff
@@ -161,6 +171,35 @@ def clean(self) -> Dict[str, Any]:
             if "domain" in cleaned_data and cleaned_data["domain"].endswith("tjhsst.edu"):
                 self.add_error("domain", "Only administrators can add tjhsst.edu domains")
 
+        # Check if this domain CNAMEs to the proper value
+        if "domain" in cleaned_data and cleaned_data["domain"] and not self.user_is_superuser:
+            try:
+                # Attempt to resolve this domain (check for CNAME records)
+                result = dns.resolver.resolve(
+                    cleaned_data["domain"], "CNAME", raise_on_no_answer=False
```

Comment:
From glancing through dnspython's documentation, it looks like you don't need `raise_on_no_answer=False` because you handle `dns.resolver.NoAnswer` later. Am I missing something, or is this just left over from previous versions of the code?

Reply:
That was left over, I forgot about that. Removed.